content/dstu2/person.md

```diff
@@ -25,12 +25,12 @@ Search for Persons that meet supplied query parameters:
  `identifier` | Yes, or `_id`                                     | [`token`]  | The person identifier.  Example: `urn:oid:2.16.840.1.113883.3.13.6|01022228`
 
 Notes:
- 
+
 - `identifier` value must include both a system and a code. Example: `identifier=urn:oid:2.16.840.1.113883.3.13.6|URN:CERNER:...:PI98N2FK5TN`
 
 ### Response
 
-<%= headers 200, GET: '[...]/Person?identifier=urn:oid:2.16.840.1.113883.3.13.6|URN:CERNER:IDENTITY-FEDERATION:REALM:AC193C1A-9763-45E1-9FA2-C1C3AA4BA16D:PRINCIPAL:R53HZ5ZL8ZX' %>
+<%= headers 200, GET: '[...]/Person?identifier=urn:oid:2.16.840.1.113883.3.13.6|urn:cerner:identity-federation:realm:687f29dd-69dd-4de5-acb1-fd8a2241ef3a:principal:uH3IW35N5RE' %>
```

Comment:
This query is not case sensitive and in actuality this type of identifier is from an external system and is in-fact mixed case.